gdi/get-data-in/application/java/configuration/advanced-java-otel-configuration.rst

```diff
@@ -110,15 +110,11 @@ The following settings control trace exporters and their endpoints:
    * - Environment variable
      - Description
    * - ``OTEL_TRACES_EXPORTER``
-     - Trace exporter to use. You can set multiple comma-separated values. For example, ``otlp,console_span``. The default value is ``otlp``. To select the Jaeger exporter, use ``jaeger-thrift-splunk``. |br| |br| System property: ``otel.traces.exporter``
```

Comment:
LOL I think java has NEVER supported `console_span`, and I can only surmise that's copypasta from python. 🙈

Reply:
*sigh* Who knows how that got in here.

Reply:
Yeah, that might have happened. Sometimes we consolidate field descriptions and things slip through.